tests/src/app.spec.ts

```diff
@@ -44,93 +45,96 @@ describe('GET /v2/', function () {
 // https://docs.docker.com/registry/spec/api/#pulling-an-image
 access.forEach((item) => {
 	describe(`get latest manifest with valid jwt`, function () {
-		it('should respond with OK', function (done) {
-			const jwt = generateToken('', '', [item]);
-			request(app(REGISTRY_URL))
+		it('should respond with OK', async function () {
+			const jwt = generateToken([item]);
+			await request(app(REGISTRY_URL))
 				.get(`/v2/${item.alias}/manifests/latest`)
 				.set('Authorization', `Bearer ${jwt}`)
 				.send()
-				.expect(200, done);
+				.expect(200);
 		});
 	});
 
 	describe(`get sha256 digest with valid jwt`, function () {
-		it('should respond with OK', function (done) {
-			const jwt = generateToken('', '', [item]);
-			request(app(REGISTRY_URL))
+		it('should respond with OK', async function () {
+			const jwt = generateToken([item]);
+			await request(app(REGISTRY_URL))
 				.get(`/v2/${item.alias}/blobs/sha256:d3adc0de`)
 				.set('Authorization', `Bearer ${jwt}`)
 				.send()
-				.expect(200, done);
+				.expect(200);
 		});
 	});
 
 	describe(`get latest manifest with invalid jwt (missing alias)`, function () {
-		it('should respond with with 403', function (done) {
+		it('should respond with with 403', async function () {
+			// @ts-expect-error deleting a required property
 			delete item['alias'];
-			const jwt = generateToken('', '', [item]);
-			request(app(REGISTRY_URL))
+			const jwt = generateToken([item]);
+			await request(app(REGISTRY_URL))
 				.get(`/v2/${item.alias}/manifests/latest`)
 				.set('Authorization', `Bearer ${jwt}`)
 				.send()
-				.expect(403, done);
+				.expect(403);
 		});
 	});
 
 	describe(`get latest manifest with invalid jwt (missing name)`, function () {
-		it('should respond with with 403', function (done) {
+		it('should respond with with 403', async function () {
+			// @ts-expect-error deleting a required property
 			delete item['name'];
-			const jwt = generateToken('', '', [item]);
-			request(app(REGISTRY_URL))
+			const jwt = generateToken([item]);
+			await request(app(REGISTRY_URL))
 				.get(`/v2/${item.alias}/manifests/latest`)
 				.set('Authorization', `Bearer ${jwt}`)
 				.send()
-				.expect(403, done);
+				.expect(403);
 		});
 	});
 
 	describe(`get latest manifest with invalid jwt (missing access)`, function () {
-		it('should respond with with 403', function (done) {
+		it('should respond with with 403', async function () {
+			// @ts-expect-error deleting a non-existent property
 			delete item['access'];
```

Comment:
This seems like it's meant to be testing something @klutchell but none of the fixtures include an access property so this should be a no-op?

Reply:
This should be `actions` not `access` IIRC.